src/formatter.rs

```diff
@@ -68,6 +68,8 @@ where
     R: BufRead,
 {
     let mut command = String::new();
+    let mut stdout = std::io::stdout();
+    stdout.lock();
     while let Some(token) = tokenizer.next_token() {
```

Comment:
I think the lock should be passed via an argument

Reply:
Yes, maybe even a generic type that implements `io::Read`

It's just something to try out for now, maybe we don't like that version then, but I would like to see it 😅

Reply:
Alright, I'll fix these comments later today :smiley: